strapi.json

```diff
@@ -29,11 +29,11 @@
     "Search Features [Result Highlighting]": "No",
     "Search Features [Geospatial Search]": "No",
     "Search Features [Search within uploads (pdf,doc)]": "No",
-    "Image Manipulation": "No",
+    "Image Manipulation": "Yes",
     "Image Focalpoint support": "",
     "Image Facedetection": "",
-    "Optimized Image Encoding": "No",
-    "Asset fingerprinting": "",
+    "Optimized Image Encoding": "Yes",
+    "Asset fingerprinting": "Yes",
```

Comment:
I did not find information on this feature. Can you provide a link for me?

Reply:
I've set the value to "No", but what do you mean by "Asset fingerprinting" to make sure I totally understand?

Reply:
Asset fingerprinting is mainly a feature common in digital asset management systems. 
https://en.wikipedia.org/wiki/Digital_video_fingerprinting